static/js/zamboni/devhub.js

```diff
@@ -787,7 +787,7 @@ function initPayments(delegate) {
         "img/zamboni/contributions/after.png",
         "img/zamboni/contributions/roadblock.png",
     ],
-        media_url = $("body").attr("data-media-url"),
+        static_url = $("body").attr("data-static-url"),
         to = false,
         img = $("<img id='contribution-preview' alt='' />");
         moz = $("input[value='moz']");
```

Comment:
Not your code but my lint-sense is tingling - this looks like it might be global since the previous line has a ';'